Dockerfile

```diff
@@ -1,12 +1,8 @@
-FROM python:3.10.0a7-alpine3.13
+FROM alpine:3.19
```

Comment:
Alpine 3.18 includes aws-cli v2 as a 'main' package, so all you need to do is just run apk add aws-cli